pkg/identity/numericidentity.go

```diff
@@ -286,7 +286,8 @@ var (
 		labels.IDNameInit:       ReservedIdentityInit,
 		labels.IDNameRemoteNode: ReservedIdentityRemoteNode,
 	}
-	reservedIdentityNames = map[NumericIdentity]string{
+	ReservedIdentityNames = map[NumericIdentity]string{
+		IdentityUnknown:            "unknown",
```

Comment:
For reviewers: Given how we're using `ReservedIdentityNames`, it looks okay to add the unknown identity to the list (+ it is, in a sense, a reserved identity as well). If that's not okay, I can work around it in the new `Entity()` function.

Reply:
At a glance it seems like it should be OK? 

I'm a little bit wary that this moves us more in the direction of defining what an "unknown" identity is, when it really represents a lack of information rather than something specific. If we had perfect information in all monitor output messages it wouldn't exist. But I don't think this change will particularly influence this.

:+1: